garage/tf/models/gaussian_mlp_model.py

```diff
@@ -57,8 +61,13 @@ def __init__(self,
                  max_std=None,
                  std_hidden_sizes=(32, 32),
                  std_hidden_nonlinearity=tf.nn.tanh,
+                 std_hidden_w_init=tf.contrib.layers.xavier_initializer,
+                 std_hidden_b_init=tf.zeros_initializer,
                  std_output_nonlinearity=None,
-                 std_parameterization='exp'):
+                 std_output_w_init=tf.contrib.layers.xavier_initializer,
```

Comment:
please make an issue to split this model into several different ones based on architecture. there are several model architectures here:

| description | mean | std_param |
| --- | --- | --- |
| fixed variance | MLP1 | parameter (trainable or not) |
| split network | MLP1 | MLP2 |
| shared network/multi-headed |  MLP1/head1 | MLP1/head2 |

Reply:
So we will have three different GaussianMLPModels?

Reply:
at least, yes. TBD if we should also split based on the stddev parameterization

Reply:
https://github.com/rlworkgroup/garage/issues/587